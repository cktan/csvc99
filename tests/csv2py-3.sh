# Test Case : with Quotes and Escape Characters 
../csv2py -e '\' in/csv2py-3.csv 
