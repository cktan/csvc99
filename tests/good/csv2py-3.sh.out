[
	['name','age','note'],
	['John Johnny Doe','30','"Programmer"'],
	['Jane','25','"Designer"']
]

